core/base/message_window.cc

```diff
@@ -0,0 +1,101 @@
+#include "message_window.h"
+// This must be included before many other Windows headers.
+#include <windows.h>
+
+#include <cassert>
+#include <iostream>
+
+namespace foxglove {
+namespace windows {
+
+void printLastError() {
```

Comment:
nit: This should be called `PrintLastError`according to the Google C++ style guide

Reply:
updated